sdk/ml/azure-ai-ml/tests/model/e2etests/test_model.py

```diff
@@ -68,25 +69,6 @@ def test_crud_file(self, client: MLClient, randstr: Callable[[], str], tmp_path:
         # with pytest.raises(Exception):
         #     client.models.get(name=model.name, version="3")
 
-    def test_create_autoincrement(self, client: MLClient, randstr: Callable[[], str], tmp_path: Path) -> None:
-        path = Path("./tests/test_configs/model/model_no_version.yml")
```

Comment:
If this yaml file is no longer used. Can we remove it?

Reply:
Being used in the unit tests